### PR TITLE
Misc. typos

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell`

### CODE REVIEW

File: sympy/utilities/iterables.py [hunk 1]
```diff
@@ -2166,7 +2166,7 @@ def kbins(l, k, ordered=None):
     [[0, 1, 2, 3], [4]]
 
     The ``ordered`` flag which is either None (to give the simple partition
-    of the the elements) or is a 2 digit integer indicating whether the order of
+    of the elements) or is a 2 digit integer indicating whether the order of
```
Comment (luzpaz) [inline sympy/utilities/iterables.py]: please review

File: sympy/utilities/runtests.py [hunk 2]
```diff
@@ -1535,7 +1535,7 @@ class SymPyDocTestFinder(DocTestFinder):
     classmethods, and properties.
 
     Modified from doctest's version by looking harder for code in the
-    case that it looks like the the code comes from a different module.
+    case that it looks like the code comes from a different module.
```
Comment (luzpaz) [inline sympy/utilities/runtests.py]: please review

File: sympy/utilities/iterables.py [hunk 3]
```diff
@@ -2166,7 +2166,7 @@ def kbins(l, k, ordered=None):
     [[0, 1, 2, 3], [4]]
 
     The ``ordered`` flag which is either None (to give the simple partition
```
Comment (normalhuman) [inline sympy/utilities/iterables.py]: I think "which" is not needed here

File: sympy/utilities/iterables.py (hunk 1 above)
Comment (normalhuman) [reply sympy/utilities/iterables.py]: It's a strange way to encode two bits of information, but the description appears to be correct.

File: sympy/utilities/runtests.py (hunk 2 above)
Comment (normalhuman) [reply sympy/utilities/runtests.py]: This can probably presented better.

Comment (smichr) [reply sympy/utilities/runtests.py]: Perhaps:
```
Modified from doctest's version by looking harder for code when
it looks like the code comes from a different module.
```

**Pull request merged**